.golangci.yml

```diff
@@ -5,8 +5,6 @@ run:
 linters:
   enable:
     - bodyclose
```

Comment:
deadcode and depguard are unmaintained/deprecated

Reply:
Removed `depguard` because I don't know why we were using it (and it wasn't configured.) The purpose is to restrict the module from importing certain packages; we don't have such a policy for this tool as far as I know.